ci-operator/config/open-cluster-management/kube-state-metrics/open-cluster-management-kube-state-metrics-release-2.4.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.16-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.15-linux
```

Comment:
This should be go1.16 instead of go1.15. Go 1.15 is about to reach end of life.

Reply:
thanks @ray-harris 
applied your comments.